### PR TITLE
messenger: complete multi-emission response streams

### PR DESCRIPTION
**Pull request by sohkai**
Previously, if an RPC endpoint returned a multi-emission response (e.g. `api.pastEvents()`), the user would never know when it was completed, despite `@aragon/wrapper` completing the stream on its side.

It wasn't an issue before, since all of our multi-emission responses were meant to be infinite, but with `pastEvents()` this is now a problem.

This change to `@aragon/wrapper` should be backwards compatible to those who are still using `@aragon/api@v1`, since that version of the API only includes infinite multi-emission responses and all others are single-emission responses that already terminate due to a `first()`.

This also mitigates the receiving side of #229, but does not solve it completely (we would need to send an 'unsubscribe' event back to the RPC handler to mitigate it completely).

### CODE REVIEW

**Comment by sohkai** [comment]
Deferring this for later; it's unclear exactly if this is necessary and what impact it would have.

**Comment by sohkai** [comment]
It turns out `api.pastEvents()` is a single-emission result, not a multi-emission one, so the root reason behind this change is moot.

However, that being said, I think it's still useful functionality to have in the messenger layer, to cover our backs when we need a stream that does end.

File: packages/aragon-rpc-messenger/src/signals.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const COMPLETE = Symbol('RPC_COMPLETE')
```
Comment (2color) [inline packages/aragon-rpc-messenger/src/signals.js]: Could this pose problems given that symbols can't be serialised for `postMessage`?
https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm

Comment (sohkai) [reply packages/aragon-rpc-messenger/src/signals.js]: Ah yes, this was designed around that; the receiver actually imports this symbol from `@aragon/rpc-messenger` and uses it directly within their context, so it never gets sent through a `postMessage`.

**Comment by sohkai** [comment]
Superceded by #367.

**Pull request merged**